pyro/infer/svgd.py

```diff
@@ -71,6 +71,8 @@ def __init__(self, bandwidth_factor=None):
         """
         :param float bandwidth_factor: Optional factor by which to scale the bandwidth
         """
+        if bandwidth_factor is not None:
```

Comment:
Remove the check after switching to the setter below.

Reply:
done